operations/rates.md

```diff
@@ -116,9 +233,11 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `IsActive` | boolean | required | Whether the rate is still active. |
 | `IsEnabled` | boolean | required | Whether the rate is currently available to customers. |
 | `IsPublic` | boolean | required | Whether the rate is publicly available. |
+| `Type` | [Rate type](rates.md#rate-type) | required | Type of the rate |
 | `Name` | string | required | Name of the rate. |
```

Comment:
When adding rates, I notice that the `Name` is specified as `localized text`, but not here in the response. Is that correct? In that case, is the name the one for the default language code used at the enterprise? It seems a bit odd that `Name` and `ShortName` are strings, but `ExternalNames` and `Description` are `Localized text`.

Reply:
It returns the first name